src/modules/likes.js

```diff
@@ -0,0 +1,36 @@
+const url = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/7P9rWHY7WsKk1rSNM8nF/likes';
+
+export const getLikesNumber = async () => {
```

Comment:
- [ ] I saw your presentation. In the video all features working as expected. However, I was not able to find like button. When I run the app in local. Make sure your like button is displaying as expected. Due to this reason I was not able to test your like button's functionality.

Your image            |  Original image
:-------------------------:|:-------------------------:
<img width="1440" alt="Screenshot 2023-01-03 at 18 29 21" src="https://user-images.githubusercontent.com/69011963/218203193-7ea9c231-79ec-4358-8a1b-36d2d06da9c6.png"> | <img src="https://user-images.githubusercontent.com/69011963/218203300-c4d02f01-0275-47ee-9d72-f470e2c0100a.png" width="1440"/>

Reply:
Hello @Rustamxon7,
Thank you for the review. We will work on the changes. However, we would like a little more information about how you ran the app locally so we can identify where the problem might be. On our end, the like buttons are displaying and even so in the live demo link, so it's a bity difficult to troubleshoot.

Reply:
Hi @Thoth1111, 
I run your app with `npm start`. Based on your documentation in the readme file I was not able to find the correct command to run it. I strongly suggest you add guidelines in Readme. Thank you!

Reply:
Okay, thank you. We use npm start as well. Perhaps we can request a re-review and see if the issue persists. We've removed the commented code and I'll also update the Readme now.